src/pages/profile/style.css

```diff
@@ -0,0 +1,122 @@
+/* :root { */
+/*     --color-blue: #000046; */
+/*     --color-blue1: #64648c; */
+/*     --color-blue2: #96a0be; */
+/*     --color-blue5: #e6ebf5; */
+/*     --color-blue7: #f5faff; */
+/*     --color-green: #64dc78; */
+/*     --color-offwhite: #f0f5fa; */
+/*     --color-lightgrey: #f5f5f5; */
+/* } */
```

Comment:
Why have you commented these styles out? They are being used throughout the CSS files

Reply:
Corrected, however, this was pulled from the global style sheet.